### PR TITLE
Created simple appliaction

### LINKED ISSUE #5: Putting it all together
Use `index.js` to take in arguments, like files or something, and print out the result.

### PR DESCRIPTION
**Pull request by gvela024**
Closes #5

### CODE REVIEW

**Comment by gvela024** [comment]
This _should_ be good enough for Scott assuming he is willing to read some JSON.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - "4.2.x"
```
Comment (gvela024) [inline .travis.yml]: Trying this out.

**Comment by gvela024** [comment]
Travis is AWESOME!!

**Comment by JohnLe01** [comment]
:O what's Travis?

**Comment by JohnLe01** [comment]
Oh cool... Travis Ci 👍! The output looks prettier than Jenkins lol. Would be awesome if it was called Jarvis or something :)

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - "node"
```
Comment (JohnLe01) [inline .travis.yml:3]: Hahaha just "node" :P

**Comment by JohnLe01** [comment]
Cool... just tested out the simple application and it works! It prints out both the object returned from DataParser and the object returned from Translator so Scott can look at both if he wanted to. Awesome job, man! 👍

**Comment by gvela024** [comment]
You too! Alright, merging this sucker.

**Pull request merged**